Bump up Yorkie to v0.3.0

cozitive opened the pull request:
<!--  Thanks for sending a pull request! -->

#### What this PR does / why we need it?
Bump up Yorkie to 0.3.0
- SDK documentation should reflect Text-Richtext merge and Counter interface change.

#### Any background context you want to provide?


#### What are the relevant tickets?
<!--
*Automatically closes linked issue when PR is merged.
Usage: `Fixes #<issue number>`, or `Fixes (paste link of issue)`.
-->
Fixes #54

### Checklist
- [x] Added relevant tests or not required
- [x] Didn't break anything

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/yorkie-team/yorkie-team.github.io?pullRequest=55) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/yorkie-team/yorkie-team.github.io?pullRequest=55) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/yorkie-team/yorkie-team.github.io?pullRequest=55) it.</sub>

jeonjonghyeok commented:
@hackerwins please check~

hackerwins commented:
@jeonjonghyeok Thanks for your contribution. 👍 
This was merged by #56.

cozitive committed:
Apply changes in Text and Counter to SDK document

SDK documentation should reflect Text-Richtext merge and
Counter interface change.

chacha912 commented:
Looks good! 

Could you also update the example code on the Products page?
<img width="798" alt="image" src="https://user-images.githubusercontent.com/81357083/215675968-0dfbda1f-0215-4b84-92c6-3de86fa7a9bc.png">

cozitive committed:
Apply changes in Counter to Products page

cozitive committed:
Bump up Yorkie to 0.3.0

hackerwins commented on MAINTAINING.md:
```diff
@@ -0,0 +1,9 @@
+
+### Updating and Deploying homepage
+
+1. Update `version` in `package.json`.
```
```suggestion
1. Update `version` in [package.json](https://github.com/yorkie-team/yorkie-team.github.io/blob/main/package.json#L36).
```

hackerwins commented on MAINTAINING.md:
```diff
@@ -0,0 +1,9 @@
+### Updating and Deploying homepage
+
+1. Update `version` in `package.json`.
+2. Update `NEXT_PUBLIC_YORKIE_VERSION` and `NEXT_PUBLIC_YORKIE_JS_VERSION` in `.env`.
```
```suggestion
2. Update `NEXT_PUBLIC_YORKIE_VERSION` and `NEXT_PUBLIC_YORKIE_JS_VERSION` in [.env](https://github.com/yorkie-team/yorkie-team.github.io/blob/34e382b81029c8cfb865bc549bdfe8a4cdd884b8/.env#L2-L3).
```

hackerwins reviewed (approved):
Thanks for your contribution.

hackerwins committed:
Apply suggestions from code review
